lua/astrocommunity/motion/harpoon/init.lua

```diff
@@ -7,38 +7,44 @@ require("astronvim.utils").set_mappings(maps)
 return {
   {
     "ThePrimeagen/harpoon",
+    branch = "harpoon2",
     dependencies = {
       "nvim-lua/plenary.nvim",
       "nvim-telescope/telescope.nvim",
     },
     cmd = { "Harpoon" },
```

Comment:
```suggestion
    cmd = { "Harpoon" },
```
The command Harpoon isn't exposed by the plugin